### DISCUSSION TITLE
Geoid height EGM96 to WGS84 for several pixels

### DISCUSSION THREAD

**Discussion by ID-work** [pyproj4/pyproj#869] [Q&A]
Hello i'm trying to calculate the geoid height in several points.
for example in lat\lon 57.8,-5.11 the height should return around 55 [m]
for 30,-50 i should get around -17 [m]

i wish to insert several lat/lon points like the ones above and get their geoid height.

can anyone help with that?

thanks

**Comment by snowman2** [comment]
See: https://github.com/pyproj4/pyproj/issues/630
